_Week8/group1/README.md

```diff
@@ -0,0 +1,32 @@
+# Group 1, Week 8 Assignment
+# Shaily Acharya, Sylvia Brown, Neel Desai
+
+
+## Overview
+
+Our simulation exercise simulates a sample dataset that can be used to estimate an individual's salary based on a variety of characteristics. In the sample model, the determinants of salary are GPA, undergraduate major, state in which their university is located, and university attended. Ultimately, the main outputs are 3 graphs that depict key characteristics of the dataset,  with the main graph depicting the number off clusters versus standard terror of the effect of GPA on salary. These graphs, along with a .csv file of the dataset, are included in this document. 
+
+## simulate.do
+
+The simulation do file defines the variables and the data generating processes that are behind our main output. 
+
+The parameter that we vary is the number of strata in our sample. In this case, the number of strata corresponds to the number of states that we are sampling from.  We generate the state-level fixed effects using a chi squared distribution— the rationale behind this is purely because we wanted to practice using a variety of different distributions in the randomization processes of our assignment. Then, we uniformly randomize the number of universities within each state, and generate the university-level fixed effects using a normal distribution. Lastly, we uniformly randomize the number of students per university.
```

Comment:
Does the total number of observations stay fixed as the number of strata changes? Be precise if your description of the data. Introduce state-level fixed effects in the prior paragraph? What does "uniformly randomize" mean? Can you describe the parameters more specifically?